tests/functional/routes/test_account.py

```diff
@@ -188,3 +188,61 @@ def test_change_password_with_incorrect_password(
         assert res.status_code == 403
         assert user.check_password("abc123") is False
         password_changed_email.assert_not_called()
+
+    @mock.patch("app.services.user.send_email_change_confirm_email.queue")
+    def test_change_email_with_correct_credentials(
```

Comment:
We don't allow this action for un-authenticated accounts - so I don't think it's necessary to pass two fields into this, you can just let authenticated users change their email by PATCHing `/account`

Reply:
Okay I've changed it so that it sends only the new password